x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/utils/import_rules_utils.ts

```diff
@@ -190,7 +181,7 @@ export const importRules = async ({
                     language,
                     license,
                     machineLearningJobId,
-                    outputIndex: signalsIndex,
+                    outputIndex: '',
```

Comment:
This field is deprecated, no? We just need to remove it at some point, right?

Reply:
@spong Yep, good point. We left it because the feature flag was still there and so it may have still been needed. I can follow up to remove it!